### PR TITLE
Setup logistic regression

### PR DESCRIPTION
**Pull request by yutanagano**
* First attempt at setting up logistic regression machinery
* Used native LogisticRegression and MultiOutputRegressor classes
* Works fine, but do we want to go a more manual approach to have more control over the fitting process etc?

### CODE REVIEW

**Comment by yutanagano** [comment]
## We should move to pytorch
- Sklearn does not allow for custom loss functions, so if we want to do our funky "use OLGA as our regularisation" type stuff, then we will have to move to pytorch
- On the bright side in the spirit of moving fast, while I set up the pytorch-based custom model we can definitely throw together a prototype using sklearn

**Review by isabella-sodi — APPROVED** [review]
Looks good to me
